step-templates/roundhouse-database-migration.json

```diff
@@ -55,6 +55,17 @@
           "Octopus.ControlType": "SingleLineText"
         }
       },
+      {
+        "Id": "8363dfa2-4509-4c57-9784-5baf27e96397",
+        "Name": "roundhouseAuthenticaitonMethod",
```

Comment:
Spelling: `roundhouseAuthenticaitonMethod` should be `roundhouseAuthenticationMethod`